mesonpy/__init__.py

```diff
@@ -192,11 +188,15 @@ def _has_extension_modules(self) -> bool:
         # Assume that all code installed in {platlib} is Python ABI dependent.
         return bool(self._wheel_files['platlib'])
 
+    @property
+    def normalized_name(self) -> str:
+        return self._project.name.replace('-', '_')
```

Comment:
Not related to editable installs: shouldn't this also map `"."` to `"_"`?

Reply:
Good catch! But not really here, though I should definitely be doing that in the module name.